chart/openfaas/templates/jetstream-queueworker-dep.yaml

```diff
@@ -79,6 +79,8 @@ spec:
           value: "{{ .Values.jetstreamQueueWorker.logs.format }}"
         - name: upstream_timeout
           value: "{{ .Values.gateway.upstreamTimeout }}"
+        - name: backoff
```

Comment:
What happens if empty string is inserted instead, i.e. just whatever we see in `.Values.queueWorkerPro.backoff`?

Do we not need a change to the dedicated queue-worker chart too?

Reply:
The queue-worker will use a default of `exponential` when the env variable is empty. I guess we could leave out the default.

Reply:
This has now been updated and the backoff value has also been added to the dedicated queue-worker chart.